packages/sveltekit/README.md

```diff
@@ -222,37 +222,26 @@ For [row level security](https://supabase.com/docs/learn/auth-deep-dive/auth-row
 ```ts
 // src/routes/profile.ts
 import { supabaseServerClient, withApiAuth } from '@supabase/auth-helpers-sveltekit';
-import type { RequestHandler } from './__types/profile';
+import type { RequestHandler } from './__types/protected-route';
 
 interface TestTable {
 	id: string;
 	created_at: string;
 }
 
 interface GetOutput {
-	user: User;
-  data: TestTable[];
+	data: TestTable[];
 }
 
-export const get: RequestHandler<GetOutput> = async ({ locals }) =>
-	withApiAuth(
-		{
-			redirectTo: '/',
-			user: locals.user
-		},
-		async () => {
-      const { data } = await supabaseServerClient(session.accessToken)
-        .from<TestTable>('test')
-        .select('*');
-
-			return {
-				body: {
-					user: locals.user,
-          data
-				}
-			};
-		}
-	);
+export const get: RequestHandler<GetOutput> = async ({ locals, request }) =>
+	withApiAuth({ user: locals.user }, async () => {
+		const { data } = await supabaseServerClient(request).from<TestTable>('test').select('*');
+
+		return {
+			status: 200,
```

Comment:
Do we need to return the `200` status? I think this is returned implicitly if we do not add a different status code.